front-end-poc-pmml-linear-model-viewer/src/LinearRegressionViewAdaptor.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import * as PMML from "./generated/www.dmg.org/PMML-4_4";
-import { Line, LinearRegressionView } from './LinearRegressionView';
+import { Line, Range, LinearRegressionView } from './LinearRegressionView';
+import { monitorEventLoopDelay } from 'perf_hooks';
```

Comment:
Unused import here?

Reply:
Yep. I'll remove. 

Please let me know when you're done with this PR and I'll do all the fixes at once.